seatunnel-connectors/seatunnel-connector-spark-jdbc/src/main/scala/org/apache/seatunnel/spark/sink/Jdbc.scala

```diff
@@ -16,47 +16,98 @@
  */
 package org.apache.seatunnel.spark.sink
 
-import scala.collection.JavaConversions._
-
+import org.apache.commons.collections.CollectionUtils
 import org.apache.seatunnel.common.config.CheckConfigUtil.checkAllExists
 import org.apache.seatunnel.common.config.CheckResult
 import org.apache.seatunnel.shade.com.typesafe.config.ConfigFactory
-import org.apache.seatunnel.spark.SparkEnvironment
+import org.apache.seatunnel.spark.{JdbcConfigs, SparkEnvironment}
 import org.apache.seatunnel.spark.batch.SparkBatchSink
-import org.apache.spark.sql.{Dataset, Row}
 import org.apache.spark.sql.execution.datasources.jdbc2.JDBCSaveMode
+import org.apache.spark.sql.{Dataset, Row}
+import org.slf4j.{Logger, LoggerFactory}
+
+import java.sql.{Connection, DriverManager, Statement}
+import scala.collection.JavaConversions._
 
 class Jdbc extends SparkBatchSink {
 
+  private val LOG: Logger = LoggerFactory.getLogger(classOf[Jdbc])
+
+  private var jdbcConfigs: JdbcConfigs = _
+
+  private def getConnection(): Connection = {
+    Class.forName(jdbcConfigs.driver)
+    val connection = DriverManager.getConnection(jdbcConfigs.url, jdbcConfigs.user, jdbcConfigs.password)
```

Comment:
It seems that connection will be created about 3 times at worst, can connection be reused in the same job?